packages/mcp-docs-server/src/index.ts

```diff
@@ -1,27 +1,117 @@
 import fs from 'node:fs/promises';
-
-import { FastMCP } from 'tylerbarnes-fastmcp-fix';
+import { Server } from '@modelcontextprotocol/sdk/server/index.js';
+import { StdioServerTransport } from '@modelcontextprotocol/sdk/server/stdio.js';
+import { CallToolRequestSchema, ListToolsRequestSchema } from '@modelcontextprotocol/sdk/types.js';
+import { z } from 'zod';
+import { zodToJsonSchema } from 'zod-to-json-schema';
 
 import { prepare } from './prepare-docs/prepare.js';
-import { blogTool } from './tools/blog.js';
-import { changesTool } from './tools/changes.js';
-import { docsTool } from './tools/docs.js';
-import { examplesTool } from './tools/examples.js';
+import { blogTool, blogInputSchema } from './tools/blog.js';
+import { changesTool, changesInputSchema } from './tools/changes.js';
+import { docsTool, docsInputSchema } from './tools/docs.js';
+import { examplesTool, examplesInputSchema } from './tools/examples.js';
```

Comment:
```suggestion
import { blogTool, blogInputSchema } from './tools/blog;
import { changesTool, changesInputSchema } from './tools/changes';
import { docsTool, docsInputSchema } from './tools/docs';
import { examplesTool, examplesInputSchema } from './tools/examples';
```

Reply:
these were here from before the package was bundled with tsup, definitely safe to remove now